packages/haho/CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Change Log
+
```

Comment:
If we need this i think would be better change version in package.json to 0.2.0 :))